bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartGraphs.java

```diff
@@ -18,6 +18,7 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.LinkedHashMap;
+import java.util.Map;
 
```

Comment:
This review is for the whole slideout, so it is not specific to this line.

First of all you have implemented a separate UI when custom data are availale, just the way I wanted 👍 

...but there are some issues